app/views/papers/_form.html.erb

```diff
@@ -53,6 +53,16 @@
     <%= f.text_field :tag_list, value: paper.tag_list.join(",") %>
   </div>
 
+  <div class="field">
+    <%= f.label :methodology_list, 'Methodologies (the following methodologies are accepted, please separate with a comma: meta-analysis, systematic review, randomized control trial, quasi-experiment, cohort, case-control, cross-sectional survey, case report)' %>
```

Comment:
multi select checkboxes is prob the best interim solution.  I'm hoping in
the long run, we can use tag boxes that you can add and remove on the show
page.  Like stackoverflow:

[image: Inline image 1]

On Tue, Aug 9, 2016 at 7:17 PM, Brian Kung <notifications@github.com> wrote:

> In app/views/papers/_form.html.erb
> <https://github.com/briankung/projectcredo/pull/49#discussion_r74167690>:
>
> > @@ -53,6 +53,16 @@
> >      <%= f.text_field :tag_list, value: paper.tag_list.join(",") %>
> >    </div>
> >
> > +  <div class="field">
> > +    <%= f.label :methodology_list, 'Methodologies (the following methodologies are accepted, please separate with a comma: meta-analysis, systematic review, randomized control trial, quasi-experiment, cohort, case-control, cross-sectional survey, case report)' %>
>
> [image: screen shot 2016-08-09 at 7 16 46 pm]
> <https://cloud.githubusercontent.com/assets/2836167/17537882/dbc701fa-5e65-11e6-82a9-71dc14a53c02.png>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/49/files/ddc284b119a291513400a37b5e5a8c2366d63154#r74167690>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACKsZEyEmBz4GYIjjeRFP9--meTGA1bhks5qeRiDgaJpZM4JflY4>
> .
>